openlibrary/tests/solr/test_update_work.py

```diff
@@ -354,6 +356,74 @@ def test_author_info(self):
         assert d['author_facet'] == ['OL1A Author One', 'OL2A Author Two']
         assert d['author_alternative_name'] == ["Author 1"]
 
+    LCC_TESTS = {
```

Comment:
```suggestion
    # {'Test name': (doc_lccs, solr_lccs, sort_lcc_index)}
    LCC_TESTS = {
```

Reply:
I've never seen that syntax before; what does this do?

Reply:
Oh, it's just a comment to outline the data's shape :P 👍 Lgtm!